Gemfile

```diff
@@ -1,8 +1,14 @@
 source 'https://rubygems.org'
 
+gem 'capybara'
+gem 'pg'
+gem 'sinatra'
+gem 'thin'
```

Comment:
Oooh, what's this gem?

Reply:
Thin? You know, I can't fully remember. I will try and find out where it was suggested for web app working and send you the link